src/templates/default/styles/index.css

```diff
@@ -96,7 +107,8 @@
   object-fit: cover;
   vertical-align: middle;
   object-position: center;
-  filter: grayscale(100%);
+  filter: grayscale(5%);
```

Comment:
I was wondering we don't need this filter style at all! It is better to have a natural user photo. Thanks for this change.